sdk/src/main/java/io/dapr/client/DaprPreviewClientBuilder.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright 2021 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.client;
+
+import io.dapr.config.Properties;
+import io.dapr.serializer.DaprObjectSerializer;
+import io.dapr.serializer.DefaultObjectSerializer;
+import io.dapr.v1.DaprGrpc;
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+
+import java.io.Closeable;
+
+public class DaprPreviewClientBuilder {

```

Comment:
There is significant overlap between this builder and the existing DaprClientBuilder. I propose we add a new build method in the existing builder instead: buildPreviewClient().

Another option is to refactor the existing builder into a new abstract class and have both builder classes to inherit from it.

Reply:
Added a new method `buildPreviewClient`